src/ai/ai.cpp

```diff
@@ -413,6 +423,44 @@ void ShipAI::runOrders()
             owner->orderRoaming();  //Nothing to dock, just fall back to roaming.
         }
         break;
+
+        case AI_Repair:            //Repair at the nearest station
+        if (owner->docking_state == DS_NotDocking)
+        {
+            P<SpaceStation> friendly_station = searchFriendlyStation(15000.0);
```

Comment:
Yup, if a ship has either a jump or warp core, it should be able to seek further, like 100U.